### ISSUE TITLE
feature request: convert to both m4b and mp3

### ISSUE THREAD

**Issue by tsolignani** [openaudible/openaudible#1206]
I generally prefer m4b over mp3, but for some use cases I would need mp3. It would be handy if I could convert both to m4b and mp3 as a default and / or if there would be a right-click option to convert a single book to mp3, even it already converted to m4b. Thank you.

**Comment by openaudible** [comment]
We try to keep the UI simple. Most users do not want two converted copies of each book, so it could cause problems (and a lot of wasted disk space) if we added the option and people used it by mistake. For now, you need to move the m4b, convert to mp3, and move the m4b back.. Yes, a little bit of a pain.

**Comment by tsolignani** [comment]
Maybe you could just enable a «right click» option in books list with «convert to mp3». The right click ones are for advanced users only and, should you do a mistake, you end up with just one mp3 and not the whole library. As a matter of fact, I don't need to convert the entire library myself, but just certain books I have to listen to with not m4b supporting devices or situations... Thank you and have a nice day.

**Comment by seelerche** [comment]
Hallo, I found this issue since I have trouble converting to mp3. My preferences were set to convert to m4b, but now, I need a single book as mp3 (want to use a tiny mp3-player during sports that doesn´t support m4b), and I don´t manage to convert. I moved the m4b file out of the library and initiated a new download after having set the preferences to mp3-conversion. The result: The book was again added, but only as aax, and neither converted to m4b or mp3. There is still an arrow in front ot the title, this hasn´t changed for hours, even after restarting computer and app. I also removed another title entirely (via "delete") and added it anew, but when I had the preferences set to mp3, it was the same thing with the arrow remaining in front and no conversion happening. What can I do to get titles as mp3?
